DESCRIPTION

```diff
@@ -6,6 +6,16 @@ Authors@R:
            comment = c(ORCID = "YOUR-ORCID-ID"))
 Description: What the package does (one paragraph).
 License: GPL (>= 3)
+Imports: 
```

Comment:
@Andrea-Havron-NOAA and @msupernaw, do we need to keep both Imports and Linkingto sections?

Reply:
Both are needed as they are used for different purposes:

[LinkingTo](https://r-pkgs.org/src.html?q=inst/include#cpp-import): adds files from the other packages include folder and is needed to import C++ code
[Imports](https://r-pkgs.org/namespace.html?q=Imports#imports): makes sure the other package is installed when FIMS is installed

Reply:
Thanks @Andrea-Havron-NOAA !